build_tools/cmake/build_docs.sh

```diff
@@ -58,10 +58,12 @@ cp docs/IREE-Architecture.svg ${BUILD_DIR}/doc/
 cp docs/roadmap.md ${BUILD_DIR}/doc/
 cp docs/roadmap_design.md ${BUILD_DIR}/doc/
 
+cp docs/developer_overview.md ${BUILD_DIR}/doc/
+
 mkdir -p ${BUILD_DIR}/doc/GetStarted/
-cp docs/getting_started_windows_bazel.md ${BUILD_DIR}/doc/GetStarted/
-cp docs/getting_started_windows_cmake.md ${BUILD_DIR}/doc/GetStarted/
-cp docs/getting_started_windows_vulkan.md ${BUILD_DIR}/doc/GetStarted/
-cp docs/getting_started_linux_bazel.md ${BUILD_DIR}/doc/GetStarted/
-cp docs/getting_started_linux_cmake.md ${BUILD_DIR}/doc/GetStarted/
-cp docs/getting_started_linux_vulkan.md ${BUILD_DIR}/doc/GetStarted/
+cp docs/GetStarted/getting_started_windows_bazel.md ${BUILD_DIR}/doc/GetStarted/
```

Comment:
This directory at the moment only contains new docs? I think we can just copy everything with `*` there?

Reply:
Hmm.. we could. I don't mind the explicit list with so few items though.

Reply:
Fine for me either way. :)